src/index.html

```diff
@@ -1,15 +1,21 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>AmyWebapp</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="manifest" href="manifest.json">
```

Comment:
These relative links need a `src/` in front of them. I too thought the relative path starts from the folder the file is in, but it actually begins with the root path of the project